src/tiledb/cloud/file/ingestion.py

```diff
@@ -10,54 +8,13 @@
 from tiledb.cloud._common import utils
 from tiledb.cloud.array import info
 from tiledb.cloud.dag.mode import Mode
+from tiledb.cloud.file import udfs as file_udfs
+from tiledb.cloud.file import utils as file_utils
 from tiledb.cloud.utilities import as_batch
 from tiledb.cloud.utilities import get_logger_wrapper
-from tiledb.cloud.vcf.ingestion import find_uris_udf
 
 DEFAULT_RESOURCES = {"cpu": "1", "memory": "2Gi"}
 DEFAULT_FILE_INGESTION_NAME = "file-ingestion"
-DEFAULT_BATCH_SIZE = 100
-
-
-def sanitize_filename(fname: str) -> str:
-    """
-    Sanitizes a filename by removing invalid characters.
-
-    :param fname: A filename to sanitize
-    :return str: The sanitized string
-    """
-    name, suffix = os.path.splitext(fname)
-    name = re.sub(r"[^\w\s-]", "", name)
-    name = re.sub(r"[-_\s]+", "_", name).strip("-_")
-    return name + suffix
-
-
-def chunk_results_udf(
```

Comment:
Has the move of these functions to another module been tested? Does it work properly with `cloudpickle`?

Reply:
We will test this in a review app soon

Reply:
@ktsitsi We have tested the whole process in a review app without issues with pickling (or other issues thus far).